lib/metasploit/framework/login_scanner/pfsense.rb

```diff
@@ -0,0 +1,118 @@
+require 'metasploit/framework/login_scanner/http'
+
+module Metasploit
+  module Framework
+    module LoginScanner
+
+      # This is the LoginScanner class for dealing with Netgate pfSense instances.
+      # It is responsible for taking a single target, and a list of credentials
+      # and attempting them. It then saves the results.
+      class PfSense < HTTP
+        DEFAULT_PORT         = 443
+        LIKELY_PORTS         = [443]
```

Comment:
Can it run on port 80 etc? 👀

Reply:
For visibility:
I removed this line of code from the pfSense scanner, as it can also come from the base HTTP scanner class with the following definition:
```
DEFAULT_PORT = 80
LIKELY_PORTS = [80, 443, 8000, 8080]
```
Although the default port _is_ 443 with HTTPS enabled, so let me know if you would like me to revert the change, and have for example, the following:
```
DEFAULT_PORT = 443
LIKELY_PORTS = [80, 443]
```